src/nlr/tests/Test_WsLayerElimination.h

```diff
@@ -0,0 +1,839 @@
+
+#ifndef MARABOU_TEST_WSLAYERELIMINATION_H
+#define MARABOU_TEST_WSLAYERELIMINATION_H
+
+
+/**********************/
+/*! \file Test_WsLayerElimination.h
+ ** \verbatim
+ ** Top contributors (to current version):
+ **   Guy Amir
+ ** This file is part of the Marabou project.
+ ** Copyright (c) 2017-2019 by the authors listed in the file AUTHORS
+ ** in the top-level source directory) and their institutional affiliations.
+ ** All rights reserved. See the file COPYING in the top-level source
+ ** directory for licensing information.\endverbatim
+ **
+ ** [[ Add lengthier description here ]]
+
+**/
+
+#include <cxxtest/TestSuite.h>
+
+#include "../../engine/tests/MockTableau.h" // TODO: fix this
```

Comment:
Do the todo :)

Reply:
copied from another test:) It's fine in ours (I think)